index.bs

```diff
@@ -1019,13 +1019,11 @@ Issue: Define how this works.
 
 ## Reference ## {#data-types-reference}
 
-<dfn>RemoteReference</dfn> represents a remote reference to an exising ECMAScript object in
-[=object id map=].
-
-A [=BiDi session=] has an <dfn>object id map</dfn>. This is a weak map
+Each EMCAScript [=Realm=] has a corresponding <dfn>object id map</dfn>. This is a strong map
```

Comment:
Should this be a strong map? That feels like it would be part of a later change, not this one. We don't want to leave the spec in a state where we're creating a strong reference to every serialized object.

Reply:
move back **weak** instead of **strong**.